<launch>
  <!-- <param name="/hri/safe_distance" value="7.5"/>
  <param name="/hri/obs_size" value="2.5"/> -->

  <node name="roscausal_human" pkg="roscausal_human" type="human_state.py" output="screen">
    <param name="people_topic" type="string" value="/ped/control/teleop_persons"/>
    <param name="goal_param" type="string" value="/hri/human_goal"/>
    <param name="source_frame" type="string" value="map"/>
    <param name="target_frame" type="string" value="map"/>
  </node>

</launch>